### PR TITLE
Flash disappears

### PR DESCRIPTION
**Pull request by sigu**
Title of the PT story: Make flash messages disappear after 10 - 15 seconds

Link to PT story: https://www.pivotaltracker.com/story/show/131060533

Description of the PR:

1. Make the flash messages disappear after 15 seconds
2. Flash messages are now closable

Ready for review!
@tochman @diraulo

### CODE REVIEW

**Comment by tochman** [comment]
Please investigate the decreasing test coverage.

**Comment by sigu** [comment]
@tochman how about the naming conventions, are the names okay for the new files I have created?

**Comment by tochman** [comment]
I have no problem with the naming. Good job.

File: app/assets/javascripts/flash_messages.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// set flash messages timeout to 5 seconds
```
Comment (diraulo) [inline app/assets/javascripts/flash_messages.js]: Comment says something and the code does something else 🤔

File: app/assets/javascripts/flash_messages.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+setTimeout(function(){
+    $('#flash-message').hide()
+
```
Comment (diraulo) [inline app/assets/javascripts/flash_messages.js]: kill this line..

Comment (sigu) [inline app/assets/javascripts/flash_messages.js]: Which line exactly?

Comment (tochman) [inline app/assets/javascripts/flash_messages.js]: The empty one.

**Pull request merged**